build.sbt

```diff
@@ -22,7 +22,8 @@ val stubbornArtifact  = "com.krux"                  %% "stubborn"
 
 lazy val commonSettings = Seq(
   scalacOptions ++= Seq("-deprecation", "-feature", "-Xlint"), // , "-Xfatal-warnings"),
-  scalaVersion := "2.13.8",
+  // fix https://nvd.nist.gov/vuln/detail/CVE-2022-36944
```

Comment:
don't need to have a comment if you are not overriding anything. also can you also update other dependencies too?